tests/kernel/common/src/errno.c

```diff
@@ -47,15 +47,22 @@ static void errno_thread(void *_n, void *_my_errno, void *_unused)
 	int n = POINTER_TO_INT(_n);
 	int my_errno = POINTER_TO_INT(_my_errno);
 
+	printk("ANDY %s:%d\n", __func__, __LINE__);
```

Comment:
debug code

Reply:
Yeah, stray "-a" in the git commit sucked in my whole tree and not the file I was trying to push.  :)  Should be fixed.